service/service.go

```diff
@@ -55,47 +77,48 @@ type Settings struct {
 // Service represents the implementation of a component.Host.
 type Service struct {
 	buildInfo            component.BuildInfo
-	config               *Config
 	telemetry            *telemetry.Telemetry
 	telemetrySettings    component.TelemetrySettings
 	host                 *serviceHost
 	telemetryInitializer *telemetryInitializer
 }
 
-func New(set Settings) (*Service, error) {
+func New(ctx context.Context, set Settings, cfg ConfigService) (*Service, error) {
```

Comment:
Changes to New and Settings are not breaking since these 2 were made public this cycle.